src/main/java/org/folio/spring/controller/TenantController.java

```diff
@@ -1,53 +1,90 @@
 package org.folio.spring.controller;
 
+import static org.apache.commons.lang3.BooleanUtils.isTrue;
+import static org.springframework.http.ResponseEntity.noContent;
+import static org.springframework.http.ResponseEntity.notFound;
+import static org.springframework.http.ResponseEntity.ok;
+import static org.springframework.http.ResponseEntity.status;
+
+import java.sql.ResultSet;
+import javax.validation.Valid;
 import liquibase.exception.LiquibaseException;
+import lombok.RequiredArgsConstructor;
 import lombok.extern.log4j.Log4j2;
 import org.folio.spring.FolioExecutionContext;
 import org.folio.spring.liquibase.FolioSpringLiquibase;
 import org.folio.tenant.domain.dto.TenantAttributes;
 import org.folio.tenant.rest.resource.TenantApi;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.jdbc.core.JdbcTemplate;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
-import javax.validation.Valid;
-
 @Log4j2
 @RestController("defaultTenantController")
 @RequestMapping(value = "/_/")
 @ConditionalOnMissingBean(name = "folioTenantController")
+@RequiredArgsConstructor
 public class TenantController implements TenantApi {
+  private static final String DESTROY_SQL = "DROP SCHEMA IF EXISTS %1$s CASCADE; DROP ROLE IF EXISTS %1$s";
+  private static final String EXIST_SQL = "SELECT EXISTS(SELECT 1 FROM pg_namespace WHERE nspname=?)";
 
   private final FolioSpringLiquibase folioSpringLiquibase;
-
   private final FolioExecutionContext context;
-
-  @Autowired
-  public TenantController(FolioSpringLiquibase folioSpringLiquibase,
-                          FolioExecutionContext context) {
-    this.folioSpringLiquibase = folioSpringLiquibase;
-    this.context = context;
-  }
+  private final JdbcTemplate jdbcTemplate;
```

Comment:
Hmm.. Do we really want to use JDBC template in the controller directly?

Reply:
Moved to a service class.